include/rosparam_shortcuts/rosparam_shortcuts.h

```diff
@@ -106,6 +110,249 @@ bool convertDoublesToEigen(const std::vector<double>& values, Eigen::Isometry3d&
  */
 void shutdownIfError(const std::size_t error_count);
 
+/**
+ * @brief Parameter is not dynamic
+ * @return results is always false
+ */
+rcl_interfaces::msg::SetParametersResult static_parameter_validation(const rclcpp::Parameter& /*unused*/);
+
+/**
+ * @brief Parameter is dynamic
+ * @return results is always true
+ */
+rcl_interfaces::msg::SetParametersResult always_accept(const rclcpp::Parameter& /*unused*/);
+
+/** @brief Parameters for a node */
+class NodeParameters
```

Comment:
Maybe this should be in it's own header file called node_parameters.hpp ?

Reply:
My counter argument is that it would be nice for the user to just have to include one header. But let me know, you make the final call, and I can move it.

Reply:
This header could include the new `node_parameters.hpp` file.  That way the user doesn't have to include both.